frontend/packages/ceph-storage-plugin/integration-tests/tests/ocp-tests/installFlow.scenario.ts

```diff
@@ -31,7 +31,7 @@ const testNodeLabel = (node) => {
   expect(hasOCSTaint(nodeJSON) || !hasTaints(nodeJSON)).toBe(true);
 };
 
-describe('Testing OCS Cluster Creation', () => {
+xdescribe('Testing OCS Cluster Creation', () => {
```

Comment:
We should open a JIRA issue or Bugzilla bug to track reenabling. (Generally we put this in this title of the disabled test to make it easy to find.)